apps/mark-scan/frontend/src/apimachine_config.test.tsx

```diff
@@ -27,7 +27,7 @@ test('machineConfig is fetched from api client by default', async () => {
   apiMock.expectGetElectionState({
     precinctSelection: ALL_PRECINCTS_SELECTION,
   });
-  render(<App reload={jest.fn()} apiClient={apiMock.mockApiClient} />);
```

Comment:
This change and others below it are a result of merging main (specifically https://github.com/votingworks/vxsuite/pull/4781) into this branch and then rebasing. Sorry for the clutter :/